inspect_activations/convergence_comparision/README.md

```diff
@@ -0,0 +1,35 @@
+### convergence_check.py
+
+convergence_check.py[] file runs the neural net for activation function combinations ```relu_relu_relu, relu_sigmoid_relu, 
+relu_sigmoid_tanh ``` in the three layers with 50 epochs and plots the training and the test errors for each epoch.
+
+
+### Result:
```

Comment:
@rajasekharmekala could you please remove both the `Result` section, we can discuss it on gitter.